src/components/dropdown/index.coffee

```diff
@@ -26,19 +26,24 @@ positionDropdown = ({alignments, selection, dropdown, useScroll}, {matchWidth})
   zIndex = parentZIndex(selection.node(), true)
 
   # calculate the position of the dropdown
-  {x, y} = calculateDropdownPosition(
-    alignments,
+  {x, y, direction} = calculateDropdownPosition(
+    dropdown.attr('data-direction') or align,
```

Comment:
Storing the `data-direction` means the dropdown doesn't 'jump around'.

This solves an issue specific to things like autocomplete pickers.

- Open autocomplete picker with 10 items near the bottom of the page (i.e with space for only 5 items to show in the dropdown)
    - Dropdown shown above button
- Enter value in the filter to reduce size of dropdown (so it would fit below)
    - Dropdown jumps to below selection

After the change, the dropdown will continue to display above the selection (unless there isn't room, in which case it would jump below the selection again)